### ISSUE TITLE
[feature request] Use SIM of original communication, when replying SMS or placing the call from missed list

### ISSUE THREAD

**Issue by kwah-gh** [ubports/ubuntu-touch#1161]
### Description of the feature
As a messaging / phone app user of a dual SIM device I would like to reply to SMS or place a call from a missed call list using **by default** the SIM card, where the incoming message or missed call originated from.

Currently, there is no option to enable this mode of operation. One can set a specific SIM as default for certain action (messaging or outgoing call) or has an option that SW will ask every time to select a specific SIM.

Personally, I find proposed behavior intuitive and good to be proposed even as behavior configured by default.

**Comment by james-lyon** [comment]
I agree. It's also slightly related to https://github.com/ubports/ubuntu-touch/issues/958
